### ISSUE TITLE
runtime org.gnome.Platform branch 42 is end-of-life, please migrate to a more recent (and supported) one

### ISSUE THREAD

**Issue by antistress** [flathub/net.sourceforge.liferea#22]
Hi,
I'm note sure if this bug is useful (I'm pretty sure you already know that and probably just lack time to do the stuff)
But in case no one has already pay attention, I'd like to mention that net.sourceforge.liferea is the only package on my system to be based upon runtime org.gnome.Platform branch 42 [1]

> Info: runtime org.gnome.Platform branch 42 is end-of-life, with reason:
>    The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.
> Info: applications using this runtime:
>    net.sourceforge.liferea

which means I can't delete runtime org.gnome.Platform branch 42 to save space until net.sourceforge.liferea migrate to a newer runtime

Thanks !

(I'm running net.sourceforge.liferea 1.14.5)

--

[1] List of all flatpak installed on my system :
app.drey.EarTag                           0.4.2
com.github.unrud.VideoDownloader          0.12.5
com.mattjakeman.ExtensionManager          0.4.2
com.transmissionbt.Transmission           4.0.3 
net.scribus.Scribus                       1.5.8
net.sourceforge.liferea                   1.14.5
net.sourceforge.mcomix                    2.1.1
org.audacityteam.Audacity                 3.3.3
org.avidemux.Avidemux                     2.8.1
org.filezillaproject.Filezilla            3.64.0 
org.frozen_bubble.frozen-bubble           2.213
org.gimp.GIMP                             2.10.34
org.gnome.DejaDup                         44.2
org.gnome.Epiphany                        44.4
org.gnome.Shotwell                        0.32.1
org.libreoffice.LibreOffice               7.5.4.2
org.mozilla.Thunderbird                   102.13.0
org.mozilla.firefox                       115.0.1
org.pitivi.Pitivi                         2022.06
org.videolan.VLC                          3.0.18

**Comment by bbhtt** [comment]
Liferea depends on webkit2gtk-4.0 and libsoup2. Runtimes >42 don't have that. So we can't update.

It was just recently ported to use newer webkitgtk and libsoup3 but there is no released version with that and it has bugs.

We can update the runtime once there is a release with newer webkitgtk and libsoup3

**Comment by antistress** [comment]
Thanks for the explanation (and work) !

**Comment by bbhtt** [comment]
Stable ie. the 1.14.x (latest 1.14.6) line on Flathub stable was updated to the 44 runtime by bundling webkit2gtk-4.0 and libsoup2. This increases the app size, but should solve the annoyance of keeping around the old runtime.

Beta releases ie. the 1.15.x releases were updated to 1.15.1 which includes the port I talked about. So it's also now on runtime 44. 1.15.x will continue to stay on beta for a while until the bugs mentioned in the release notes and other issues with the port are ironed out.

Feel free to try out beta and give feedback. Set up the beta Flathub repo https://discourse.flathub.org/t/how-to-use-flathub-beta/2111 and install it as `flatpak install flathub-beta net.sourceforge.liferea`


Thanks!
